server/routes/api/choropleth.py

```diff
@@ -174,10 +173,11 @@ def choropleth_geo():
             # Process Statistical Observation if valid.
             if ('data' in population_by_geo.get(geo_id, [])
                     and population_by_geo[geo_id]['data']):
```

Comment:
Is this check necessary given that `get_data` can handle empty input?

Reply:
thank you. you're right, it's not necessary, if 'data' is not in payload, the value will be skipped because get_data returns None.
So check is not necessary